index.html

```diff
@@ -0,0 +1,121 @@
+<!DOCTYPE html>
+<html lang="ko">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Linkbrary</title>
+  <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+  <header>
+    <div class="header-container">
+      <div class="header-logo-login">
+        <a href="./index.html"><img src="./images/head_logo.svg"></a>
+        <a id="login" href="./signin.html">로그인</a>
+      </div>
+  </header>
+  <main>
+    <div class="title">
+      <h1 class="info-title"><span class="info-title-accent">세상의 모든 정보</span>를<br>쉽게 저장하고 관리해 보세요</h1>
```

Comment:
`br`태그는 셀프 클로징을 해주면 좋을거 같네요
```suggestion
      <h1 class="info-title"><span class="info-title-accent">세상의 모든 정보</span>를<br/>쉽게 저장하고 관리해 보세요</h1>
```

Reply:
추가로 `<br />`은 줄바꿈의 용도로는 사용을 잘 안 하는 편인데요. 왜 그런지 찾아보셔도 좋을거 같습니다 ㅎㅎ